lib/utilities/reexport-template.js

```diff
@@ -1,7 +1,12 @@
 /* global define */
-define("{{DEST}}", ["{{SRC}}", "ember", "exports"], function(__index__, __Ember__, __exports__) {
-  "use strict";
-  __Ember__["default"].keys(__index__).forEach(function(key){
+define('{{DEST}}', ['{{SRC}}', 'ember', 'exports'], function(__index__, __Ember__, __exports__) {
+  'use strict';
+  var keys = Object.keys || __Ember__['default'].keys;
+  var forEach = Array.prototype.forEach && function(array, cb) {
```

Comment:
oops, even local tests caught it. I should have let them finish before asking for a review. Should be fixed now